eng/common/scripts/Verify-ChangeLog.ps1

```diff
@@ -11,6 +11,11 @@ Set-StrictMode -Version 3
 
 . (Join-Path $PSScriptRoot common.ps1)
 
+# Set ChangeLogLocation for specific package
+if ($PackageName -eq 'io.clientcore:annotation-processor') {
```

Comment:
Don't modify `/eng/common` scripts, it isn't allowed unless it was a change being ported from `azure-sdk-tools` repo.

Also, I don't fully understand why this is needed, mind explaining what this is doing.